src/rf/rf/settings/base.py

```diff
@@ -288,7 +288,7 @@ def get_env_setting(setting):
 AWS_WORKSPACE_BUCKET = environ.get('RF_WORKSPACE_BUCKET')
 AWS_EMR_INSTANCES = environ.get('RF_EMR_INSTANCES')
 AWS_EMR_CLUSTER_NAME = environ.get('RF_EMR_CLUSTER_NAME')
-AWS_EMR_DEBUG = environ.get('RF_EMR_DEBUG')
+AWS_EMR_DEBUG = bool(environ.get('RF_EMR_DEBUG'))
```

Comment:
Still seems to evaluate the string to `True`:

```bash
vagrant@app:~$ cat /etc/rf.d/env/RF_EMR_DEBUG
False
vagrant@app:~$ envdir /etc/rf.d/env/ python
Python 2.7.6 (default, Jun 22 2015, 17:58:13)
[GCC 4.8.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.environ['RF_EMR_DEBUG']
'False'
>>> bool(os.environ['RF_EMR_DEBUG'])
True
```

Maybe you want `ast.literal_eval`?

```bash
>>> import ast
>>> ast.literal_eval(os.environ['RF_EMR_DEBUG'])
False
```